### PR TITLE
Fix docs and minor stuff + bump version before release

### PR DESCRIPTION
**Pull request by piotmag769**
Closes #

## Introduced changes

-
-

## Checklist

- [ ] Linked relevant issue
- [ ] Updated relevant documentation
- [ ] Added relevant tests
- [ ] Performed self-review of the code
- [ ] Added changes to `CHANGELOG.md`

### CODE REVIEW

**Review by Arcticae — APPROVED** [review]

**Review by Radinyn — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,29 +26,35 @@ To run tests scripts, you have to install:
 Before creating a contribution, make sure your code passes the following checks
 
 ```shell
-./scripts/test_forge.sh
-./scripts/test_cast.sh
```
Comment (war-in) [inline CONTRIBUTING.md:30]: Why it is removed? Those scripts have some nice features like downloading correct Scarb and Cairo compiler versions.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,29 +26,35 @@ To run tests scripts, you have to install:
...
-cargo fmt --check
-cargo lint
+$ cargo test
```
Comment (THenry14) [inline CONTRIBUTING.md:29]: I think we should include `scripts/prepare-for-tests.sh` here, otherwise cast tests may fail :(

Comment (piotmag769) [reply CONTRIBUTING.md:29]: It is mentioned above

Comment (THenry14) [reply CONTRIBUTING.md:29]: Ahh yes, sorry didn't check what is above, only saw the changes. You're right, that should suffice.

**Review by THenry14 — APPROVED** [review]

**Pull request merged**